docs/01-programs/02-candy-machine/09-available-guards/07-gatekeeper.md

```diff
@@ -4,21 +4,24 @@ description: "Restricts minting via a Gatekeeper Network e.g. Captcha integratio
 
 import { Accordion, AccordionItem } from '/src/accordion.jsx';
 
-# Gatekeeper
+# Gatekeeper / Civic Pass
 
 ## Overview
 
-The **Gatekeeper** guard checks whether the minting wallet has a valid **Gateway Token** — also known as a **Civic Pass** — from a specified **Gatekeeper Network**.
+The **Gatekeeper** guard checks whether the minting wallet has a valid **Gateway Token** — also known as a **Civic Pass**. There are various types of Civic Pass you can use to guard your mint, ranging from completing a simple Captcha challenge, all the way to capturing a 3D video selfie that guarantees a "one NFT per human" mint. 
 
-In most cases, this token will be obtained after completing a Captcha challenge but any Gatekeeper Network may be used.
+ You select the type of Civic Pass to use by configuring the corresponding [**Gatekeeper Network**](https://docs.civic.com/civic-pass/overview/how-it-works#the-on-chain-gatekeeper-network) in the Gatekeeper guard.
 
-There isn’t much to set up on the Candy Machine side but, depending on the selected Gatekeeper Network, you may need to ask the minting wallet to perform so pre-validation checks to grant them the required Gateway Token.
```

Comment:
Not a big fan of this change as the transition with the previous paragraph is a bit harsh. May I suggest the following instead?


> In order to add a **Gatekeeper** to your Candy Machine, you must first select a **Gatekeeper Network** which defines how user verification is performed.
> 
> There isn’t much to set up on the Candy Machine side, you simply need to provide the public key of the Gatekeeper Network of your choice and whether or not Gateway Tokens should expire after being used. Note that depending on the selected Gatekeeper Network, you may need to ask the user to perform so pre-verification checks to grant them the required Gateway Token.
> 
> Here is a list of available Gatekeeper Networks and how to get started with them.